ibm/service/vpc/data_source_ibm_is_security_group.go

```diff
@@ -81,6 +82,12 @@ func DataSourceIBMISSecurityGroup() *schema.Resource {
 							Description: "Security group id: an IP address, a CIDR block, or a single security group identifier",
 						},
 
+						isSgRuleLocal: {
```

Comment:
Why schema here is a string and next datasource a list ?

Reply:
This response is as part of security group and other is part of security group rule response hence the difference